src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -235,7 +238,8 @@ public void Start()
 
                 if (_repl != null) {
                     WriteLog.To.Sync.I(Tag, $"{this}: Starting");
-                    Native.c4repl_start(_repl);
+                    Native.c4repl_start(_repl, reset);
+                    Config.Options.Reset = reset;
```

Comment:
This won't hurt anything, but is unneeded.

Reply:
Removed